index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script  src="js/index.js"></script>
+    <meta charset="UTF-8">
+    <title>Nelson and Ahmed</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
```

Comment:
I think there are a lot of spaces left throughout your code so they should be closed up. The above is one of them.